### PR TITLE
Add MessageHandler Class

### PR DESCRIPTION
**Pull request by matteeyao**
## User Story
**As a** user
**I want** the UI to ask for a user's input and respond to that input
**So that** I can choose game mode, place mark, etc.

### Main Menu
**Given** I am a user
**When** I launch the terminal TTT app
**Then** I should be greeted
**AND** be provided w/ a menu of game modes
**AND** be asked to choose from one of the game modes
**AND** if input is invalid, I should be asked to choose an option again

### Request for Board Dimension
**Given** I am a user
**When** I choose a game mode
**Then** I should be provided w/ a message asking about the size of the board I would like to play on
**AND** in input is invalid, I should be asked to enter board size again

### Request for `Player`(s)' `mark`(s)
**Given** I am a user
**When** I start up a game of TTT
**Then** I should be asked to provide my emoji mark
**AND** if there is another human player, she should be asked to provide her emoji

### Print Board
**Given** I am a user
**When** I am playing a game of TTT
**Then** I should be shown the current state of the board w/ corresponding marks

### Request for `Player`'s move
**Given** I am a user
**When** I begin a game of TTT
**Then** I should be asked to input my move 1-9
**AND** if input is invalid, notify me and ask for input again
**AND** if position is taken, notify me and ask for input again

### Notifies End-game Results
**Given** I am a user
**When** a game of TTT is over
**Then** I should be notified of the winner 
**OR** if the game ends in a draw

### CODE REVIEW

File: App/UI/MessageHandler.cs [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        }
+
+        public static void PrintRequestToChooseGameMode()
+        {
+            Console.Write("Choose from one of the above options: ");
+        }
+
+        public static void PrintRequestToChooseGameModeAfterInvalidInput()
+        {
+            Console.Write("Invalid option. Choose again from options 1-3: ");
```
Comment (solidlikeplato) [inline App/UI/MessageHandler.cs:30]: This should match up with current code functionality: i.e. shouldn't reference other modes that don't exist yet and are commented out on lines 18, 19

File: App/UI/MessageHandler.cs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        
+        public static void PrintRequestToInputBoardSizeAfterInvalidInput()
+        {
+            Console.Write("Invalid board size. Enter board size 3, 4, or 5: ");
+        }
+        
+        public static void PrintRequestForPlayerOnesMarker(bool isOpponentComputer)
+        {
+            string title = isOpponentComputer ? "your" : "player one's";
+            Console.Write($"Enter {title} mark (Hit enter to default to \u274C): ");
```
Comment (solidlikeplato) [inline App/UI/MessageHandler.cs:46]: maybe extract default marks to variable. I had to look up what symbols \u274C and \u2B55 were

File: App/UI/MessageHandler.cs [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        }
+
+        public static void PrintRequestForPlayerTwosMarker()
+        {
+            Console.Write("Enter player two's mark (Hit enter to default to \u2B55): ");
```
Comment (solidlikeplato) [inline App/UI/MessageHandler.cs:51]: This class seems to Console.Write are a lot of static messages using different methods. maybe simplify by extracting the messages to an enum and making one (or only a few) method to write that takes the message as input

Comment (matteeyao) [reply App/UI/MessageHandler.cs]: ✅

**Pull request merged**